stakingModel/contracts/SimpleDAIStaking.sol

```diff
@@ -56,11 +58,15 @@ contract SimpleDAIStaking is DSMath, Pausable, SchemeGuard {
         address _dai,
         address _cDai,
         address _uniswap,
-        address _fundManager
+        address _fundManager,
+        uint256 _blockInterval
+
     ) public SchemeGuard(Avatar(address(0))) {
         dai = ERC20(_dai);
         cDai = cERC20(_cDai);
         uniswap = _uniswap;
+        blockInterval = _blockInterval;
+        lastUBICollection = block.number + blockInterval;
```

Comment:
this prevents first interval withdraw, probably should be just block.number
need to add a unit test